modules/wb67-can-rs485.sh

```diff
@@ -36,6 +36,8 @@ hook_module_init() {
 		sysfs_gpio_direction $GPIO_CAN_EN out
 		sysfs_gpio_set $GPIO_CAN_EN 0
 	fi;
+
+	systemctl restart wb-mqtt-mbgate.service
```

Comment:
не, так не совсем правильно. В других скриптах есть примеры того, как быть со связанными сервисами. Тебе тут надо сделать
```suggestion
	hook_once_after_config_change "service_restart wb-mqtt-mbgate"
```

так правильно привязываться к изменениям в конфиге, чтобы почём зря сервис не дёргать, а ещё туда включена обёртка `service_restart`, которая, например, не перезапускает сервис при загрузке системы, когда это и не нужно.

В соседних файлах здесь есть примеры использования этой магии, можно подсмотреть